app/helpers/transaction_type_helper.rb

```diff
@@ -38,4 +38,13 @@ def answer_for_transaction_type(legal_aid_application:, transaction_type:)
       t("generic.none")
     end
   end
+
+  def regular_transaction_answer_by_type(legal_aid_application:, transaction_type:)
+    regular_transaction = legal_aid_application.regular_transactions.where(transaction_type:).first
```

Comment:
For a single return you can use `find_by`
```suggestion
    regular_transaction = legal_aid_application.regular_transactions.find_by(transaction_type:)
```

Reply:
Should an application be able to have more than one regular transaction for a given category?

If the answer is no, then I think we should probably be validating that and adding a unique index to the database.

That doesn't need to be a part of this pull request though.

If the answer is yes, then we should be explicit about _which_ regular transaction we want to find.

What do you think?

Reply:
sure, but the same applies to cash transactions, at least. Feel free to add a tech debt ticket or similar.